.idea/artifacts/Gradle___test___test_1_0_SNAPSHOT_war__exploded_.xml

```diff
@@ -0,0 +1,42 @@
+<component name="ArtifactManager">
```

Comment:
AAAAA why .idea in the pull request???

Reply:
i don't know, what is a problem. gitignore ignored all folders except .idea. Do you know, what could it be?

Reply:
Done